src/languages/en-US.ts

```diff
@@ -327,6 +330,25 @@ export default class extends Language {
 		commandClearDescription: 'Clears the queue list.',
 		commandClearDenied: `${REDCROSS} You can't execute this command when there are over 4 members! You must be a dee-jay or a moderator!`,
 		commandClearSuccess: ({ amount }) => `${REDCROSS}  Pruned ${amount} ${amount === 1 ? 'song' : 'songs'}.`,
+		commandExportQueueDescription: 'Exports your queue to a `.squeue` file.',
+		commandExportQueueExtended: {
+			extendedHelp: [
+				"Have a queue you liked and want to replay later? Or maybe you want to send your friends what you're listening right now",
+				"Use `exportqueue` and I'll pack the music in your queue into a neat file you can either save or share with your friends!",
+				'When you want to play it back, just use it with `play`, `add` or `importqueue`!'
+			],
+			multiline: true
+		},
+		commandExportQueueSuccess: ({ guildName }) => `${GREENTICK} Here's the current queue for ${guildName}!`,
+		commandImportQueueDescription: 'Imports a queue saved as an `.squeue` file.',
```

Comment:
```suggestion
		commandImportQueueDescription: 'Imports a queue saved as a `.squeue` file.',
```

"saved as a dot squeue file". If you say the `.` as "dot" then it's "a" and not "an", which I think is correct here. May be worth checking how MSFT programs show this on Windows or something. In ES you wrote `a .squeue` btw.